pimpmycause/pimpmycause/urls.py

```diff
@@ -17,16 +17,16 @@
 from django.contrib import admin
 
 from core.views import HomepageView, ContactView, AboutView, MeetTheTeam, OurPartners, Partnerships, BecomeAPartner, WhoWeAre, HowItWorks
-from profiles.views import LoginView, RegistrationView, RegistrationComplete, ActivationComplete, profile_update
+from profiles.views import RegistrationView, RegistrationComplete, ActivationComplete, profile_update, logout_view
 from profiles.forms import PimpUserRegistrationForm
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^$', HomepageView.as_view(), name="index"),
-    url(r'^login/', LoginView.as_view(), name="login"),
+    url(r'^logout/', logout_view, name="logout_view"),
```

Comment:
Maybe this should be like : url(r'^logout/', LogoutView.as_view(), name="logout"),
to match the others? If so L20 above needs updating and L39 from views.py.